src/sentry/templates/sentry/emails/organization-invite-request.html

```diff
@@ -0,0 +1,15 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+  <h3>Request for Access</h3>
+
+  <p><strong>{{ inviter.get_salutation_name }}</strong> has request for <strong>{{ email }}</strong> to join the <a href="{{ organization_link }}">{{ organization.name }}</a> organization.</p>
```

Comment:
how about `has requested to invite [email] to the [name] organization`

Reply:
Yeah sounds good